frontend/test/unit/test-utils/api-service-mock.ts

```diff
@@ -62,9 +62,9 @@ export const mockCreateApiService = (
   applyMocks: (mockAdapter: MockAdapter) => void
 ): void => {
   let apiService: apiService.ApiService | undefined
-  afterEach(() => {
-    apiService = undefined
-  })
+  // afterEach(() => {
+  //   apiService = undefined
+  // })
```

Comment:
If the tests pass, just delete it :+1: 

I don't remember why I put this here and can't tell what it would meaningfully do :shrug: